src/g_canvas.h

```diff
@@ -189,6 +191,7 @@ struct _glist
     unsigned int gl_loading:1;      /* am now loading from file */
     unsigned int gl_willvis:1;      /* make me visible after loading */
     unsigned int gl_edit:1;         /* edit mode */
+    unsigned int gl_snaptogrid:1;   /* snap to grid mode */
```

Comment:
move this after `unsigned int gl_isclone:1`

Reply:
Done thanks